python/pyxet/pyxet/util.py

```diff
@@ -12,6 +12,16 @@
 MAX_CONCURRENT_COPIES = threading.Semaphore(32)
 CHUNK_SIZE = 16 * 1024 * 1024
 
+class CopyUnit:
```

Comment:
redundant? there is one in the file_operations.py

Reply:
Good catch.